dev/devicelab/bin/tasks/run_release_test.dart

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:path/path.dart' as path;
+
+import 'package:flutter_devicelab/framework/adb.dart';
+import 'package:flutter_devicelab/framework/framework.dart';
+import 'package:flutter_devicelab/framework/utils.dart';
+
+void main() {
+  task(() async {
+    final Device device = await devices.workingDevice;
+    await device.unlock();
+    final Directory appDir = dir(path.join(flutterDirectory.path, 'dev/integration_tests/ui'));
+    await inDirectory(appDir, () async {
+      final Completer<Null> ready = new Completer<Null>();
+      print('run: starting...');
+      final Process run = await startProcess(
+        path.join(flutterDirectory.path, 'bin', 'flutter'),
+        <String>['run', '--release', '-d', device.deviceId, 'lib/main.dart'],
+        isBot: false, // we just want to test the output, not have any debugging info
+      );
+      final List<String> stdout = <String>[];
+      final List<String> stderr = <String>[];
+      int runExitCode;
+      run.stdout
+        .transform(utf8.decoder)
+        .transform(const LineSplitter())
+        .listen((String line) {
+          print('run:stdout: $line');
+          stdout.add(line);
+          if (line.contains('To quit, press "q".'))
+            ready.complete();
+        });
+      run.stderr
+        .transform(utf8.decoder)
+        .transform(const LineSplitter())
+        .listen((String line) {
+          print('run:stderr: $line');
+          stdout.add(line);
+        });
+      run.exitCode.then((int exitCode) { runExitCode = exitCode; });
+      await Future.any<dynamic>(<Future<dynamic>>[ ready.future, run.exitCode ]);
+      if (runExitCode != null)
+        throw 'Failed to run test app.';
```

Comment:
'Failed to run test app; runner exited with exit code $runExitCode.'

Reply:
added